03-task-manager/starter/controllers/tasks.js

```diff
@@ -0,0 +1,32 @@
+const Task = require('../models/Task')
+const getAllTasks = (req, res) => {
+    res.send('All items')
+}
+
+const createTask = async (req, res) => {
+    const task = await Task.create(req.body)
+    //res.send('Create task')
+    res.status(201).json({task})
+}
+
+const getTask = (req, res) => {
```

Comment:
You need to implement these methods retrieving and updating data on MongoDB. See the example located in the final/controller directory to know how to implement this.